q2_feature_table/plugin_setup.py

```diff
@@ -18,7 +18,14 @@
     name='feature-table',
     version=q2_feature_table.__version__,
     website='https://github.com/qiime2/q2-feature-table',
-    package='q2_feature_table'
+    package='q2_feature_table',
+    citation_text=('The Biological Observation Matrix (BIOM) format or: how '
```

Comment:
I still disagree with adding the citation here because q2-feature-table doesn't seem like the ideal home for it. But given that citations will work a lot differently in the future, and this citation registration will need to be updated at that point, it doesn't seem too harmful to include.

I just want to note that one of the major goals of QIIME 2 is to track relevant citations automatically for users, and to give authors credit where credit is due. I am in full support of citing the BIOM format (obviously, it needs to be cited somewhere), but it isn't clear right now where the best home for the citation is (future citation support may make this easier to determine).